packages/create-atomic/template/package.json

```diff
@@ -4,11 +4,14 @@
   "version": "1.25.4",
   "private": true,
   "scripts": {
+    "postinstall": "npm run setup-lambda && npm run setup-cleanup",
```

Comment:
Not sure why this disappeared...

Reply:
The removal is normal and should be kept.
The create-atomic-template is considered a package of the monorepo (it helps greatly with dependencies check).
See the https://github.com/coveo/cli/blob/master/packages/create-atomic/scripts/preparePackageJsonTemplate.mjs,
The postInstall is programmatically added to the `hbs` template.

I marked with 'no' every removal that must be reverted.

Reply:
Ok makes more sense now, I had to run this script before testing locally